### PR TITLE
simple bash install

### PR DESCRIPTION
**Pull request by arcbtc**
Just creates a bash script that can be used to run and install. Has fail-safe in case someone doesnt read install guide and uses from root directory.

```
wget https://raw.githubusercontent.com/lnbits/lnbits/snapcraft/lnbits.sh &&
chmod +x lnbits.sh &&
./lnbits.sh
```

### CODE REVIEW

File: lnbits.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  if [ -d lnbits ]; then # Someone didnt follow the install guide
+    # Check out the main branch
+    git checkout main
+  else
+    # Clone the LNbits repository
+    git clone https://github.com/lnbits/lnbits.git
+    cd lnbits
+
+    # Check out the main branch
+    git checkout main
```
Comment (motorina0) [inline lnbits.sh:27]: `git checkout main` runs on both branches of the `if`. Is the `if` required?

**Review by motorina0 — APPROVED** [review]
LGTM
see comment

**Comment by prusnak** [comment]
I am wondering if this method should not be the number two method - while keeping poetry as recommended option one.

Also I think we should not install soon-to-be unsupported Python 3.9. We should just make sure that LNbits run with all supported versions (up to 3.12 - PR https://github.com/lnbits/lnbits/pull/2619 takes care of this).

**Comment by arcbtc** [comment]
> I am wondering if this method should not be the number two method - while keeping poetry as recommended option one.
> 
> Also I think we should not install soon-to-be unsupported Python 3.9. We should just make sure that LNbits run with all supported versions (up to 3.12 - PR #2619 takes care of this).

Sure, lets move to option 2, sqlalchemy seems like a big job tho and will require a good few weeks worths of work.

File: lnbits.sh (hunk 1 above)
Comment (arcbtc) [reply lnbits.sh]: fixed

**Pull request closed without merging**